legal-api/src/legal_api/models/dataclass.py

```diff
@@ -0,0 +1,33 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module holds data classes."""
+
+from dataclasses import dataclass, field
+from typing import List
+
+
+@dataclass
+class ReviewFilter:
+    """Used for filtering and sorting reviews."""
+
+    status: List[str] = field()
+    start_date: str = ''
+    end_date: str = ''
+    nr_number: str = ''
+    identifier: str = ''
+    completing_party: str = ''
+    submitted_sort_by: str = ''
+    submitted_sort_order: bool = ''
+    page: int = 1
+    limit: int = 10
```

Comment:
Can you move this into `review.py` instead of separate file. For now we don't have any thing other than model in this folder. I would have suggested to move this into `utils` or create a new folder for dataclass, since we are planning to strip out model into a separate lib while migrating to GCP its better to keep it inside